### DISCUSSION TITLE
Storing sessions in Keycloak 26 - blog post feedback

### DISCUSSION THREAD

**Discussion by mhajas** [keycloak/keycloak#35523] [Feedback]
There is a blog post on the current state of sessions and this is a room for all related questions.

Blog post link: https://www.keycloak.org/2024/12/storing-sessions-in-kc26

**Comment by razi-ahmad** [comment]
I am storing sessions, in redis. I change the infinispan to redis because, whenever I was upgrading the Keycloak version, it was dropping the sessions because, sometimes, pod didn't start properly.  This [dependency](https://github.com/luneo7/infinispan-cachestore-redis) I am using in my keycloak.
